docs/manual/docs/contributing/doing-a-release.md

```diff
@@ -242,15 +242,15 @@ with the following utilities: ***sed***, ***xmlstarlet*** and ***sftp***.
     cd /home/frs/project/g/ge/geonetwork/GeoNetwork_opensource
     # or for RC release
     cd /home/frs/project/g/ge/geonetwork/GeoNetwork_unstable_development_versions/
-    mkdir v3.12.1
-    cd v3.12.1
-    put docs/changes3.12.1-0.txt
+    mkdir v4.4.1
```

Comment:
Shouldn't this be 4.4.2?

Reply:
No, the example assumes you are creating `4.4.1`

Reply:
I thought the example was that you were creating 4.4.2? and the previous version was 4.4.1.
Maybe I'm not understanding.

Reply:
See https://github.com/geonetwork/core-geonetwork/pull/7636/files#diff-afba6fb6c27ca0266f6c91620aac83e1f176672301fd1c3236755cd435247617L19

Reply:
Apologies I saw 4.4.2 and failed to see that it was related to the next SNAPSHOT